test/k8sT/Policies.go

```diff
@@ -87,16 +87,17 @@ var _ = Describe("K8sPolicyTest", func() {
 
 	Context("Basic Test", func() {
 		var (
-			ciliumPod string
-			clusterIP string
-			appPods   map[string]string
+			ciliumPod        string
+			clusterIP        string
+			appPods          map[string]string
+			namespaceForTest = "basictest"
```

Comment:
Can we set this based on `ginkgo.CurrentGinkgoTestDescription().FullTestText`? You would probably need to replace spaces and make it lowercase, but it might help avoid test namespaces being duplicated by mistake.

Reply:
Added!